Annotation/AbstractGraphQLRelation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @author Thibault Colette <thibaultcolette06@hotmail.fr>
+ * @copyright 2018 Thibault Colette
+ */
+
+namespace Overblog\GraphQLBundle\Annotation;
+
+/**
+ * Annotation for graphql type relation.
```

Comment:
> Annotation for GraphQL type relation.

What do you think @CocoJr?

Reply:
It's used to simulate the doctrine relation (OneToMany, etc...).

For exemple you want a type User with a field role, and the field role is an other type called Role.
You can use this annotation to indicate that and create the good schema between this type.

Tell me if you want more precision :D

Reply:
Oh my bad, it was just a comment for using `GraphQL` instead of `graphql` :stuck_out_tongue:

Reply:
Ok yes you're right, just forgot :D